### PR TITLE
fix #229 - Windspeed

### LINKED ISSUE #229: Feature request - wind speed in m/s
Greetings,

I would love a feature to show wind speed in meters pr. sec.

Cheers :)

### PR DESCRIPTION
**Pull request by Ruandv**
Added new toggle button to cater for kph and ms on metric units

### CODE REVIEW

**Comment by bluecaret** [comment]
Meters per second is a metric unit, if metric units are choosen then this should convert to m/s, however if imperial options are selected this should instead convert to feet per second (ft/s) instead.

**Comment by Ruandv** [comment]
Ok so there will not be a scenario where you'd like to display km/h or mph? 
All of those values will be converted to ft/s and m/s?

**Comment by bluecaret** [comment]
Sorry for the confusion, let me try again.

When metric units are on it will use km/h
When metric units and your new wind speed option is on it will be m/s
When metric is off, it will be mph
When metric is off and wind speed units is on it will be ft/s

**Comment by Ruandv** [comment]
Aha ok 😜 understood.

**Comment by Ruandv** [comment]
https://user-images.githubusercontent.com/2345436/131392095-ee714627-178e-4d10-b719-c111b92f5bcd.mp4

**Comment by bluecaret** [comment]
Thanks a ton!
